spacelift/resource_stack.go

```diff
@@ -34,6 +44,19 @@ func resourceStack() *schema.Resource {
 			StateContext: resourceStackImport,
 		},
 
+		SchemaVersion: 1,
+
+		StateUpgraders: []schema.StateUpgrader{
+			{
+				Version: 0,
+				Type:    resourceStackResourceV0().CoreConfigSchema().ImpliedType(),
+				Upgrade: func(ctx context.Context, rawState map[string]interface{}, meta interface{}) (map[string]interface{}, error) {
+					rawState["import_state"] = ""
```

Comment:
Can we add a reasoning here? The reasoning aren't super obvious unless you read this PR's description.